recipes/goofit/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "2.2.2" %}
+{% set name = "goofit" %}
+
+{% set goofit_proc_version = "1.0.0" %}
+{% set goofit_proc_type = "cpu" if cuda_compiler_version|default("None") == "None" else "gpu" %}
+{% set goofit_cuda_supported = ["None", "9.2", "10.0"] %}
+
+package:
+  name: goofit-split
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82884fd3021555da0953776f81199cbea60287f6bca87b0ecdaa5dd212692c67
+  patches:
+    - patches/thrustorder-2.2.2.patch
+  
+build:
+  skip: true  # [win]
+  {% if cuda_compiler_version|default("None") in goofit_cuda_supported %}
```

Comment:
`in` -> `not in` and drop the else? Or is it needed to work around a rendering quirk?

Reply:
I thought so, but it seems to work now, so must have had a different issue earlier. Updated.